tests/environment.py

```diff
@@ -213,16 +213,23 @@ def _docker_wait_for_log(container: str, program: str, regex: str, start_line: i
 
 
 def _device_wait_for_log(device: None, log_path: str, regex: str,
-                         start_line: int, timeout: int) -> bool:
-    """Waits for log mathing regex expression to show up."""
-    device.sendline("tail -n10 -f {}".format(log_path))
-    match = device.expect(
-            pattern=[regex, device.pexpect.EOF, device.pexpect.TIMEOUT],
-            timeout=timeout)
-    if match == 1:
-        return True
+                         timeout_set: int, start_line: int = 0):
+    """Waits for log matching regex expression to show up."""
+    device.sendline("tail -f -n +{:d} {}".format(start_line, log_path))
```

Comment:
Tail starts counting at 1, and `tail -n +0` gives the same as `tail -n +1`.
```suggestion
    device.sendline("tail -f -n +{:d} {}".format(start_line + 1, log_path))
```

Reply:
Applied!